vueapp/src/views/Settings.vue

```diff
@@ -13,7 +13,7 @@
           <v-row>
             <v-col cols="12">
               <v-select
-                :items="voiceOptions"
+                :items="$store.state.settings.voiceOptions"
```

Comment:
Could you change this to be reading from a getter in the ...mapGetters? Keeps the Vuex store code uniformed. We try not to access state directly unless there is no way around it.